projects/ngx-formentry/src/mock/orders.json

```diff
@@ -334,5 +334,45 @@
       "type": "testorder",
       "resourceVersion": "1.10"
     }
+  ],
+  "diagnoses": [
```

Comment:
I think these should probably be separated out into it's own file. It would also be good to have some validation that this supports non-coded diagnoses (does this support non-coded diagnoses?)

Reply:
No, at this moment this development is just for coded diagnoses.

Reply:
I have also separated the files.